lms/djangoapps/email_marketing/signals.py

```diff
@@ -150,8 +150,12 @@ def email_marketing_register_user(sender, user=None, profile=None,
     if user.is_anonymous():
         return
 
+    site_domain = _get_site_domain()
+    if not site_domain:
```

Comment:
If there is no site domain, we can associate this user with the default Sailthru user list. Let's not skip a registered user from being synched on Sailthru side.